docs/Earn/liquid-staking/dot/dot-liquid-staking-mechanics.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Polkadot Staking Mechanics
+id: dot-liquid-staking-mechanics
+---
+# Polkadot Staking Mechanics
+
+DOT Liquid Staking lets the user stake their funds through the ANKR deposit address on the Polkadot network, accumulate rewards, and receive their stake+rewards when unstaking.
+
+ANKR principles for the staking are:
+
+* Receive DOT from the user to the ANKR deposit address.
+
+* Mint aDOTb on Ethereum. Make all claiming aDOTb transactions on the Ethereum network.
+
+* Use the fixed exchange rate 1:1 for aDOTb to DOT. aDOTb always represents 1 DOT token even though we sacrifice ERC-20 compatibility for this.
+
+* aDOTb ratio changes, thus all aDOTb balances “automatically” grow.
+
+Staking is well-described by its requirement, fees, rewards, validators, as well as smart contracts and workflow that shows the API endpoints to call to interact with these smart contracts.
+
+## Requirements
+
+The minimum requirements when staking are:
+
+* Minimum value to stake — 1 DOT.
+
+* Maximum value to stake — unlimited, at the user’s discretion.
+
+Staking can be repeated anytime, without waiting for the current staking to completed as staking on Polkadot is done by the ANKR-owned wallet where users send their stake directly.
+
+The minimum requirements when unstaking are:
+
+* Minimum value to unstake — 0 DOT.
+
+* Maximum value to unstake — staked amount + % of the reward, up to the user’s balance.
+
+* Release period — up to 7 days.
+
+## Fees
+
+When staking, the user pays:
+
+* 2% commission on the reward, to ANKR.
+
+When unstaking, the user pays no fees.
+
+The users must also count in the cost for outgoing transactions.
+
+## Rewards
+
+Validators/delegators receive rewards daily.
+
+ANKR backend calculates the ratio of rewards to the total stake amount weekly and publishes it to the aDOTb smart contract respectively.
+
+## Validators
+
+ANKR delegates the staked funds to trusted validators from Polkadot, which may change in 1 day or longer.
```

Comment:
what is meant by 'may change in 1 day or longer' do you mean that is how long it takes to reach trusted validators and start earning rewards?